gems/core/tasks.py

```diff
@@ -149,16 +149,17 @@ def import_contacts(self):
         logger.exception('importing contacts :: Failed to fetch contacts')
 
     count = 0
-    for contact in all_contacts:
-        try:
-            contact, created = Contact.objects.get_or_create(msisdn=contact['msisdn'])
-            if not created:
-                continue
-            contact.vkey = contact['key']
-            contact.save()
-            count += 1
-        except Exception:
-            logger.exception('creating contact :: Failed to create a contact %s' % (contact['msisdn']))
+    for item in all_contacts:
+        if 'msisdn' in item:
```

Comment:
add an else to this if that logs the item variable when it does not contain an msisdn, just for possible future debugging sessions.